### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by juancruzgs**

### CODE REVIEW

File: app/google-services.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "project_info": {
+    "project_number": "934985628718",
+    "firebase_url": "https://base-project-b3d16.firebaseio.com",
```
Comment (julianfalcionelli) [inline app/google-services.json:4]: Please invite us to this Firebase project  😉

File: app/build.gradle [hunk 2]
```diff
@@ -1,4 +1,17 @@
+buildscript {
+    repositories {
+        maven {
+            url 'https://maven.fabric.io/public'
```
Comment (julianfalcionelli) [inline app/build.gradle:4]: Are we using Fabric ? I don't remember 👀

Comment (juancruzgs) [reply app/build.gradle:4]: Firebase Crashlytics uses Fabric internally.
https://firebase.google.com/docs/crashlytics/get-started

File: app/google-services.json (hunk 1 above)
Comment (juancruzgs) [reply app/google-services.json:4]: Done

**Review by apuli — APPROVED** [review]

**Review by julianfalcionelli — APPROVED** [review]

**Pull request merged**